test/Libraries/TestServices/App.config

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<configuration>
+    <appSettings>
+        <add key="DynamoBasePath" value=""/>
+        <add key="RequestedLibraryVersion" value=""/>
```

Comment:
this shouldn't be used any longer, we should use `RequestedLibraryVersion2`

Reply:
to clarify, I am pretty sure I obsoleted `RequestedLibraryVersion` years ago and I wonder if @sm6srw is going to remove it in his PR? RequestedLibraryVersion2 supports actually versions where as RequestedLibraryVersion only supported somewhere around ASM 223 - 5...

Reply:
replaced with RequestedLibraryVersion2